doc/build_chipsec.md

```diff
@@ -14,60 +15,49 @@ A Help to build your own ChipSec and SecureBoot USB keys
 	sudo apt-get install efitools
 ~~~
 
-## Tool to build the USB keys : create-keys.sh
+## Tool to build the USB key: create-chipsec.sh
 
 > **Note:**
 Some sub scripts require access to sudo commands.
 >
 
-Both build scripts support writing to a block device (`/dev/sdc` for example)
+The script supports writing to a block device (`/dev/sdc` for example)
 or a standard file, which can be later copied to a USB drive using `dd`.
 
-### Build USB KEY 1
 Plug a new USB key (attached on /dev/sdc in this case).
 
 ~~~
-./create-keys.sh live /dev/sdc
+./create-chipsec.sh /dev/sdc
 ~~~
 
 Unplug the USB key.
 
+## Test the system
 
-### Build USB KEY 2
-Plug a new USB key (attached on /dev/sdc in this case).
-
-~~~
-./create-keys.sh shell /dev/sdc
-~~~
-
-Unplug the USB key.
-
-## Boot on keys
+Plug the key, start the computer and pick one of the two boot modes.
 
-Plug one of keys, start the computer.
-   
-### USB KEY 1
+### Live Debian distribution
 
-1. boot on the USB key, then at the bootloader prompt start linux live
+1. boot on the USB key, then at the bootloader prompt start "Debian GUN/Linux"
 1. when finished booting, login as root (no password)
+1. if you need an alternate keyboard, use eg. `loadkeys fr`.
 1. from the root terminal, launch ChipSec with "chipsec_main.py".
 1. alternately you can run the dump_system.sh script which will also gather
 information about the machine (hardware present, firmware versions et.c)
 
-### USB KEY 2 :
+### EFI Tool
 
 1. Go the BIOS/Firmware configuration and set the platform to SecureBoot
    enabled and reset to Setup Mode.
-1. either:
-    - boot on USB key and launch EFI binaries from EFI shell (Shell.efi is
-      automaticaly started).
-    - OR interrupt the normal boot to select a shell EFI from boot configuration
-      and launch EFI binaries from EFI shell.
-1. execute binaries from EFI shell :
-    - identify the USB key letter storing the binaries with commmands "fs0:" or
-      "fs1:" or fsX: ... then "dir"
-    - launch "KeyTool.efi" to import trust keys.
-1. replace (in that order) db, KEK and PK using files from keytool folder
+1. boot on USB key, then at the bootloader prompt either:
+    - start "Keytool" directly, which lets you execute binaries from its menu,
+    - OR start "EFI Shell" and launch EFI binaries from EFI shell (eg. launch
+      "EFI/keytool/KeyTool.efi" to import trust keys).
+1. from the EFI shell, you can identify the USB key letter storing the binaries
+   with commmands "fs0:" or "fs1:" or fsX: ... then "dir". Within Keytool, the
+   drives will have names instead. The USB key should be named "ESP", but you
+   can confirm it by browsing its content through the "Execute Binary" menu.
+1. With Keytool, replace (in that order) db, KEK and PK using files from "EFI/keys" folder
```

Comment:
Since fc70cc9 an empty DBX (denylist) is generated as well and thus should be added as well. Also it might be worth noting to use `DBX.esl`, `DB.esl`, `KEK.esl` **but** `PK.auth`.

Reply:
Done.